config/initializers/redis.rb

```diff
@@ -0,0 +1,16 @@
+# Redis is the storage for Sidekiq.
+# we assume defaults are fine for test/development.
+# https://github.com/mperham/sidekiq/wiki/Using-Redis
+if ENV["RACK_ENV"] == "production"
+  vcap = ENV["VCAP_SERVICES"]
+  vcap_config = JSON.parse(vcap)
+  vcap_config.keys.each do |vcap_key|
+    if vcap_key.match(/redis\d+-swarm/)
+      redis_config = vcap_config[vcap_key][0]['credentials']
+      redis_url = "redis://#{redis_config['password']}@#{redis_config['hostname']}:#{redis_config['port']}"
+      #STDERR.puts "REDIS_URL=#{redis_url}"
```

Comment:
no, but I understand why you note it. I intended to use it for testing #40 when I got to that point. I can zap it for now if you like.